sdk/src/state_machine/mod.rs

```diff
@@ -1,12 +1,11 @@
-mod env {
-    pub fn read_unchecked<T>() -> T {
-        todo!()
-    }
-}
+// mod env {
```

Comment:
let's remove the unused code?

Reply:
Good catch - I had meant to delete completely. Fixed!